utils/warnings.ml

```diff
@@ -426,6 +428,8 @@ let message = function
       Printf.sprintf "the %S attribute is used more than once on this expression" attr_name
   | Inlining_impossible reason ->
       Printf.sprintf "Inlining impossible in this context: %s" reason
+  | No_cmx_file name ->
+      Printf.sprintf "no cmx file was found in path for module " ^ name
```

Comment:
I would suggest:

Printf.sprintf
      "no cmx file was found in path for module %s, \
       and it's interface was not compiled with -opaque" name